CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Align JavaScript modules format in components ([PR #1769](https://github.com/alphagov/govuk_publishing_components/pull/1769))
+* Converts track options correctly when passed in correctly ([PR #1772](https://github.com/alphagov/govuk_publishing_components/pull/1772))
```

Comment:
Nitpick: can this and the PR name be the same?

Reply:
Done